packages/palette/src/elements/Input/Input.tsx

```diff
@@ -142,6 +154,25 @@ const StyledInput = styled.input<StyledInputProps>`
         cursor: default;
         ${INPUT_STATES.disabled}
       }
+
+      ${props.title &&
+      css`
+        ::placeholder {
+          opacity: 0;
+        }
+      `}
     `
   }};
 `
+
+const StyledLabel = styled.label<StyledInputProps>`
+  position: absolute;
+  top: 50%;
+  left: 5px;
```

Comment:
Hm, is there a way to get at our token value — I know `0.5` has some issues. `themeGet("space['0.5']")` maybe?

Reply:
I tried multiple ways, but none were successful 😞

Reply:
We should probably just introduce some named aliases for these, like `space.half-x` or something. Anyway; fine for now.

Reply:
(You also could just pass these using `attrs`)